.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## 📌 Ticket
+
+Please provide the URL to the Jira ticket related to this PR.
+
+## 📄 Description
+
+Please include a summary of the changes made in this PR.
+
+## 🚀 Demo
+
+If applicable, please add a screenshot or video to illustrate the changes.
+
```

Comment:
We have the preview link for that, no?

Reply:
yes, but a screenshot or a video really helps to illustrate the change fast so you don't have to go hunt for what could possibly have changed based on the source. I'd like to keep that point.